client/src/__tests__/unit/Restriction.test.tsx

```diff
@@ -4,24 +4,28 @@ import Restrictions, { weekDays } from "../../components/Restrictions";
 import { findTestWrapper } from "../../util/findTestWrapper";
 
 describe("Restrictions", () => {
-  const wrapper = mount(<Restrictions />);
-  const buttons = findTestWrapper(wrapper, "button");
+  // const wrapper = mount(<Restrictions />);
```

Comment:
Do we want to clean up commented lines of code?

Reply:
Yeah we do, I was a bit lazy, there was a bunch of already commented out code from before :D.

Reply:
Yeah I figured haha. Thanks for taking on this 🙏